### ISSUE TITLE
Crash involving Rendering

### ISSUE THREAD

**Issue by loadstarnes** [the-realest-stu/Rustic#56]
Heres the crash report: [https://pastebin.com/SN3ku0bQ](https://pastebin.com/SN3ku0bQ)

From what I can read, it involves JustEnoughItems and Rustic.

The crash can be consistently reproduced in my game with the mods listed in the crash report by pressing R (the recipe viewer control button for JEI) on a (vanilla Minecraft) apple.

_I have consistently reproduced this crash in my game with ONLY the mods RUSTIC, and JEI._ 
The crash report for when I isolated these two mods can be found here: [https://pastebin.com/30tWp5kc](https://pastebin.com/30tWp5kc)
This, I assume, means the problem is either in JEI, Rustic, or both.

It looks like the actual thing that couldn't render was "FluidBottleModel"...

Is this a JEI problem, or a Rustic problem? Is a quick fix available?

**Comment by DietmarKrause** [comment]
same here. Trying to fill immersive creosote oil in rustics liquid barrel leads to crash and unladeable world: https://pastebin.com/Ku8asqVk.

**Comment by DietmarKrause** [comment]
https://github.com/AllTheMods/ATM-3/issues/51

**Comment by loadstarnes** [comment]
I have just reproduced the crash by trying to fill a glass bottle with apple juice from the crushing bucket.

**Comment by DietmarKrause** [comment]
caan be: perhaps different errors . Sorry.

**Comment by loadstarnes** [comment]
@DietmarKrause I'm not saying they are *definitely* different errors. Since they both have to do with the same mod and fluid rendering, I'd say that either it's likely they are the same error, or they are *related* errors! Anyway, your crash report should help @the-realest-stu or whoever can fix this/tell us what's wrong

**Comment by the-realest-stu** [comment]
I'll look into this now. It's definitely two different crashes, as they're coming from two separate class files, but they might both be related to the way I access fluid textures. We'll see.

**Comment by the-realest-stu** [comment]
@DietmarKrause your error is unrelated, and is actually a bug in IE, which has apparently been fixed in the latest update of IE. So just wait for ATM3 to update.

@loadstarnes I'm actually not sure when I'll fix that, since it's on 1.11.2. In the meantime, using an older version of JEI should fix the problem. I, personally use version 4.3.3.269 with no problems.

**Comment by loadstarnes** [comment]
@the-realest-stu Were you not able to reproduce the glitch?
I'm not sure what item you mean by "looking at the item", which item? An empty "fluid bottle", or a fluid bottle full of olive oil? I'm able to fill a bottle with apple juice from the crushing tub, drink it, and fill it again... But in my save I haven't collected any olives yet, so if the issue is a bottle of olive oil, then I'm not sure.

I *can* tell you that by using JEI's modname filter (@rustic) I can view all items added by rustic that way just fine, but it seems it crashes in any recipe-mode of JEI, which is strange.

Don't worry yourself about fixing this asap, I'll just try downgrading JEI for now. Just thought reporting the crash would be better than ignoring it...

**Comment by the-realest-stu** [comment]
Yeah, I haven't had any issues checking recipes with olive oil, or other fluids. The fact that it only happens in the JEI crafting grid makes me think it's actually a problem in JEI

**Comment by cgiacofei** [comment]
I've noticed a similar (not sure if related issue) with rendering fluid containers involving honey. The issue only seems to occur if either `Biomes 'O Plenty` or `Apiary and Honey` are loaded.

Having fluid containers filled with Rustic honey cause a rendering error and the world becomes unloadable.
Both of the above mods have their own honey fluid they add so there may be a conflict.

My temporary solution has been to set rustic bee hive spawning to zero so I'm not tempted to try and convert combs to honey.
